vcenter_operator/phelm.py

```diff
@@ -65,13 +65,15 @@ def render(self, scope, options):
             try:
                 template = env.get_template(template_name)
                 result = template.render(options)
-                self.add(result)
+                owner = env.get_source_owner(template_name)
+                self.add(result, owner)
             except (TemplateError, YAMLError):
                 LOG.exception("Failed to render %s", template_name)
 
-    def add(self, result):
+    def add(self, result, owner):
         stream = io.StringIO(result)
         for item in yaml.safe_load_all(stream):
+            item["metadata"]["ownerReferences"] = [owner]
```

Comment:
Will this create a valid template if `owner` is `None` (which it can be)? It seems it would create

```yaml
metadata:
  ownerReferences:
    - null
```
or similar. Not sure how k8s would deal with that.

Reply:
I'll add an check for owner being `None` / falsy.